epfast-annotations/src/main/java/com/exactpro/epfast/converter/TypeConverter.java

```diff
@@ -149,7 +149,7 @@ public int stringToInt(String stringValue) {
         try {
             return Integer.parseInt(stringValue);
         } catch (NumberFormatException e) {
-            throw cantParseException(stringValue, Integer.class);
+            throw cantParseException(stringValue, int.class, e);
```

Comment:
I din't know `int.class` is now allowed, can you then please change expressions like `Integer.TYPE` to this style all over this file? And squash everything to single commit and force-push. The code is ready to be merged into master